testutil/keeper/mocks.go

```diff
@@ -564,6 +564,16 @@ func (m *MockChannelKeeper) GetChannel(ctx types0.Context, srcPort, srcChan stri
 	return ret0, ret1
 }
 
+func (m *MockChannelKeeper) GetChannelConnection(ctx types0.Context, portID, channelID string) (string, exported.ConnectionI, error) {
```

Comment:
Is this actually used?

Reply:
it is not used, but if not adding this, the test is broken. because `GetChannelConnection` is added to  `ChannelKeeper` interface. https://github.com/cosmos/interchain-security/blob/856ce9e979d5e8656d6b289f06c8f64976ffb292/x/ccv/types/expected_keepers.go#L85

I wonder if this is any API breaking?

Reply:
Then it's used under the hood. Thanks!